### DISCUSSION TITLE
How to effectively use Beacon mode?

### DISCUSSION THREAD

**Discussion by zetashift** [meow-edit/meow#588] [Q&A] [answered]
I'm coming from Helix/kakoune so this is the workflow I'm used:

- select some region
- press `s` to enter a regex that will match in your region/selection
- enter to confirm and cursors will be put in place on the match.

This specific thing where I match a word and cursors are created after every much, is it possible to do in Meow?
I know I can do things with `occur-mode` but I am too bad at Emacs to make sense of it.

Currently what I do is select a region, meow-grab, and press `b` and `w` a bunch of times until I get it right.

**Comment by Esnos33** [comment, marked as answer]
You can use `meow-visit` to select symbol, but I have problems with selecting words https://github.com/meow-edit/meow/discussions/576

**Reply by zetashift** [reply]
Yeah, this was what I was looking for! Hopefully fixing meow-visit wont be hard.

**Reply by eshrh** [reply]
It seems like there are bigger issues with meow-visit in beacon as well (no fake cursors?).

**Reply by zetashift** [reply]
Is there some way to use https://github.com/magnars/multiple-cursors.el in harmony with Meow?
